packages/forms/src/elements/FauxInput.tsx

```diff
@@ -67,3 +69,24 @@ FauxInput.propTypes = {
   disabled: PropTypes.bool,
   validation: PropTypes.oneOf(['success', 'warning', 'error'])
 };
+
+export interface IIconProps extends HTMLAttributes<HTMLElement> {
+  isHovered?: boolean;
+  isFocused?: boolean;
+  isDisabled?: boolean;
+  isRotated?: boolean;
+  children: any;
+}
+
+const StartIcon = (props: IIconProps) => <StyledTextMediaFigure position="start" {...props} />;
+const EndIcon = (props: IIconProps) => <StyledTextMediaFigure position="end" {...props} />;
+
+(FauxInput as any).StartIcon = StartIcon;
+(FauxInput as any).EndIcon = EndIcon;
+
+export default FauxInput as React.FunctionComponent<
```

Comment:
Rather than using a `default` export. Could we apply this type cast to something like `export const FauxInput`?

Reply:
There is also a standardized way of applying state component properties shown in `Accordion`. This may help with the typing situation.